### PR TITLE
add dev and testing md changes

### PR DESCRIPTION
**Pull request by newtewt**
# Description

It was a bit easier to get  https://github.com/medic/cht-core/pull/6958 into master by creating a new branch vs rebasing. 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by garethbowen — COMMENTED** [review]
Nice!

Not a blocker but consider moving the unit and integration test documentation into the TESTING.md file too. At some point we should also document/link to the scalability testing suite.

**Comment by newtewt** [comment]
@garethbowen  I moved the other testing items out of the DEVELOPMENT.md to TESTING.md. Very basic move at this point.

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+1. Set Node Interpreter is set to your node install. 
+1. Set Protractor package is set to the `<cht-core-repo>/node_modules/protractor`
+1. Optionally set the Protractor options to `--capabilities.chromeOptions.args=start-maximized --jasmine.DEFAULT_TIMEOUT_INTERVAL=120000`.
+1. Select the radio button for Test.
+1. Enter the path to the Test file Ex: `<cht-core-repo>/tests/e2e/login/login.specs.js`
+1. Enter the test name. This is a bit of a chore. IntelliJ will automatically add the regex flags for begins(`^`) of line and end of line(`$`). Protractor presents the name for matching as the Describe description followed by the It description. To run the login test for should have a title you would need to put this as your matcher. `Login tests : should have a title`. An alternative would be to select Test File and run the entire file. You can add an `x` in front of `it` to disable the ones you do not need. EX: `xit('should login`)
+1. Click ok
+1. Click the run configuration dropdown and select the protractor config. 
+1. In a terminal run `grunt e2e-deploy`   NOTE: This has to happen each time you run. 
+1. Click debug button in IntelliJ.
```
Comment (andy5995) [inline TESTING.md:92]: I notice some items in this list end with a period but some do not.

Comment (newtewt) [reply TESTING.md]: Thanks I have updated!

**Review by garethbowen — APPROVED** [review]
Rubber stamped.

**Pull request merged**